container/kvm/template.go

```diff
@@ -36,8 +36,8 @@ var kvmTemplate = `
     </console>
     <input type='mouse' bus='ps2'/>
     <input type='keyboard' bus='ps2'/>
-    <graphics type='vnc' port='-1' autoport='yes' listen='127.0.0.1'>
-      <listen type='address' address='127.0.0.1'/>
+    <graphics type='vnc' port='-1' autoport='yes' listen='localhost'>
+      <listen type='address' address='localhost'/>
```

Comment:
'localhost' looks fine according in `/usr/share/libvirt/schemas/{domaincommon,basictypes}.rng`

In testing from a KVM host, `nc -vzw3 localhost 5900` was successful, as was port forwarding 5900 with ssh for a VNC console.